src/main/java/com/spotinst/sdkjava/example/statefulnode/azure/AzureStatefulNodeExample.java

```diff
@@ -0,0 +1,849 @@
+                                           .build();
+
+        //Build Network Interfaces
+        LaunchSpecNetworkInterfacesConfiguration.Builder networkInterfacesBuilder =
+                LaunchSpecNetworkInterfacesConfiguration.Builder.get();
+        LaunchSpecNetworkInterfacesConfiguration networkInterfaces =
+                networkInterfacesBuilder.setIsPrimary(true).setSubnetName("Automation-PublicSubnet").setAssignPublicIp(true)
+                                        .setPublicIpSku("Standard").setNetworkSecurityGroup(networkSecurityGroup).build();
+        List<LaunchSpecNetworkInterfacesConfiguration> networkInterfacesList = Collections.singletonList(networkInterfaces);
+
+        //Build network
+        LaunchSpecNetworkConfiguration.Builder networkBuilder = LaunchSpecNetworkConfiguration.Builder.get();
+        LaunchSpecNetworkConfiguration network = networkBuilder.setResourceGroupName(resourceGroup).setVirtualNetworkName("Automation-VirtualNetwork")
+                                                               .setNetworkInterfaces(networkInterfacesList).build();
+
+        //Build Data Disk
+        LaunchSpecDataDisksSpecification.Builder dataDiskBuilder = LaunchSpecDataDisksSpecification.Builder.get();
+        LaunchSpecDataDisksSpecification dataDisk = dataDiskBuilder.setLun(1).setSizeGB(1).setType("Standard_LRS").build();
+        List<LaunchSpecDataDisksSpecification> dataDisks = Collections.singletonList(dataDisk);
+
+        //Build OsDisk
+        LaunchSpecOsDiskSpecification.Builder osDiskBuilder = LaunchSpecOsDiskSpecification.Builder.get();
+        LaunchSpecOsDiskSpecification osDisk =
+                osDiskBuilder.setName("Automation-OsDisk").setType("Standard_LRS").setSizeGB(33).build();
+
+        //Build Extension Protected Settings
+        LaunchSpecExtensionsProtectedSettings protectedSettings = LaunchSpecExtensionsProtectedSettings.Builder.get().setScript(
+                "IyEvYmluL2Jhc2gKZWNobyAibmlyIiA+IC9ob21lL25pci9uaXIudHh0Cg==").build();
+
+        //Build Extentions
+        LaunchSpecExtensionsSpecification.Builder extensionBuilder = LaunchSpecExtensionsSpecification.Builder.get();
+        LaunchSpecExtensionsSpecification extensions = extensionBuilder.setName("Automation-Extension").setType("customScript")
+                                                                       .setPublisher("Microsoft.Azure.Extensions").setApiVersion("2.0")
+                                                                       .setMinorVersionAutoUpgrade(true).setProtectedSettings(protectedSettings).build();
+
+        List<LaunchSpecExtensionsSpecification> extensionsList = Collections.singletonList(extensions);
+        // Build Login
+        LaunchSpecLoginSpecification login =
+                LaunchSpecLoginSpecification.Builder.get().setUserName("ubuntu").setPassword("NetApp@123@321")
```

Comment:
declare username and password as private final string like auth_token and use. We don't want these values to be in the code.